app/services/decisions/case_decision_tree.rb

```diff
@@ -95,7 +95,7 @@ def after_hearing_details
     end
 
     def ioj_or_passported
-      if Passporting::IojPassporter.new(crime_application).call
+      if change_in_financial_circumstances? || Passporting::IojPassporter.new(crime_application).call
```

Comment:
Should we add CIFC as an IoJ passport?

Reply:
I think this is a good idea @timpeat - let's prioritise as a refactor, agree with @mseedat-moj that this could kick up a load of unknowns